### ISSUE TITLE
onmouseover_fp is exploitable (in Firefox)

### ISSUE THREAD

**Issue by dmitris** [yahoo/webseclab#15]
http://127.0.0.1:8088/xss/reflect/onmouseover_fp?in=bar%22+onmouseover%3D%22alert%28123%29
produces the source:

FORM>
Homepage: <input value="bar"onmouseover="alert(123)" name="in" size="40"><BR>
<input type="submit" value="Change"></FORM>

which fires XSS in Firefox.  Need to neutralize the injection.

**Comment by dmitris** [comment]
full source (got mangled in the prevous post):

```
<FORM>
Homepage: <input value="bar"onmouseover="alert(123)" name="in" size="40"><BR>
<input type="submit" value="Change"></FORM>
```

**Comment by dmitris** [comment]
fixed by #16
